packages/client-sdk-nodejs/src/config/middleware/experimental-momento-local-test-config-middleware.ts

```diff
@@ -7,8 +7,12 @@ import {
   MiddlewareStatus,
 } from './middleware';
 
-class MomentoLocalMiddlewareRequestHandler implements MiddlewareRequestHandler {
-  constructor(private readonly metadata: MomentoLocalTestConfigMetadata) {}
+class ExperimentalMomentoLocalMiddlewareRequestHandler
+  implements MiddlewareRequestHandler
+{
+  constructor(
+    private readonly metadata: ExperimentalMomentoLocalTestConfigMetadata
```

Comment:
Can you move the field declaration to the class? Just my preference for style to align with the rest of the SDK